docs/labwc-config.5.scd

```diff
@@ -890,6 +890,24 @@ situation.
 	can be caused by *<margin>* settings or exclusive layer-shell clients
 	such as panels.
 
+## MENU
+
+```
+<menu>
+  <hideDelay>200</hideDelay>
+</menu>
```

Comment:
This is referring to the openbox config name.

Reply:
Should have run `hideDelay`. Thanks.